ui/src/dashboards/actions/index.js

```diff
@@ -11,7 +11,10 @@ import {
 import {notify} from 'shared/actions/notifications'
 import {errorThrown} from 'shared/actions/errors'
 
-import {generateNewDashboardCell} from 'src/dashboards/constants'
+import {
+  generateNewDashboardCell,
```

Comment:
unrelated to your work, but this is an unconventional name for a constant... i would expect it to be a function

Reply:
Moved these to a separate file and used a more consistent name